commcare_export/writers.py

```diff
@@ -504,15 +519,18 @@ def write_table(self, table):
         """
         table_name = table.name
         headings = table.headings
-
+        # pad the data_types to be the same length as headings so we can zip them
+        padded_data_types = table.data_types + [None] * (len(table.headings) - len(table.data_types))
+        data_type_dict = dict(zip(headings, padded_data_types))
```

Comment:
neat! it is possible to make a file where `data_types` has more values than `headings` (it would be invalid). But I think in that case it'd be fine you'd just have a `None` entry in the `dict` that would never get accessed (since there'd be no heading lookup for it ever)